docs/authentication.md

```diff
@@ -0,0 +1,77 @@
+# Authentication
+
+This document details how to configure your Helios installation to authenticate
+requests between the Helios CLI and the Helios masters/cluster.
+
+# TODOs for this document
+
+- configuration needed to setup master to require authentication
```

Comment:
How about something like `--requires-auth=STRING` where `STRING` can be a client version number to require auth after or just the string `yes`

```
--requires-auth=0.8.526
# or
--requires-auth=yes
```